core-web/libs/sdk/client/src/lib/client/content/methods/get-collection/index.ts

```diff
@@ -0,0 +1,146 @@
+import { QueryBuilder } from '../../../../query-builder/sdk-query-builder';
+import { Equals } from '../../../../query-builder/utils/lucene-syntax/Equals';
+import { CONTENT_API_URL, CONTENT_TYPE_MAIN_FIELDS } from '../../const';
+import { GetCollectionResponse, SortByArray } from '../../types';
+
+export class GetCollection {
```

Comment:
Does this means we are going to have a class for each method? 

`class GetCollection`

`class GetById`

`class GetByInode`

Reply:
That's correct